go/interpreter/lfvm/opcode.go

```diff
@@ -285,6 +285,8 @@ const (
 )
 
 var toString = map[OpCode]string{
+	DATA:    "DATA",
```

Comment:
why move these two up here ?

Reply:
To have all the LFVM-specific instructions gathered in the beginning, followed by the super instructions.